tests/projects/test_projects_cli.py

```diff
@@ -90,7 +92,7 @@ def test_run_local_conda_env():
     with open(os.path.join(TEST_PROJECT_DIR, "conda.yaml")) as handle:
         conda_env_contents = handle.read()
     expected_env_name = "mlflow-{}".format(
-        insecure_hash.sha1(conda_env_contents.encode("utf-8")).hexdigest()
+        hashlib.sha1(conda_env_contents.encode("utf-8"), usedforsecurity=False).hexdigest()
```

Comment:
```suggestion
        hashlib.sha1(conda_env_contents.encode("utf-8")).hexdigest()
```

This would make ruff complain.